test/it/mocks/MilvusMockedTest.cpp

```diff
@@ -0,0 +1,12 @@
+
+#include "MilvusMockedTest.h"
+
+void
+milvus::MilvusMockedTest::SetUp() {
+    server_.Start();
+    client_ = milvus::MilvusClient::Create();
+}
+
+void
+milvus::MilvusMockedTest::TearDown() {
+}
```

Comment:
Should call server_.Stop() here?

Reply:
Yes, I think so, we need to add it while adding testing code later